base/install_opcua.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -ex
+
+. /opt/epics/install-functions.sh
+
+lnls-get-n-unpack -l https://github.com/epics-modules/opcua/releases/download/v${OPCUA_VERSION}/IOC_opcua-${OPCUA_VERSION}_Base-${EPICS_BASE_VERSION}_debian${DEBIAN_VERSION%.*}.tar.gz
+mv binaryOpcuaIoc opcua
+
+#install_module opcua OPCUA "
+#EPICS_BASE=${EPICS_BASE_PATH}
+#"
+
+module_releases="
+OPCUA=${EPICS_MODULES_PATH}/opcua
+"
+echo "$module_releases" >> ${EPICS_RELEASE_FILE}
+
+cd opcua
+
+echo "
+EPICS_BASE=${EPICS_BASE_PATH}
+" > configure/RELEASE.local
+
+make -j${JOBS}
+
+EPICS_HOST_ARCH=$(${EPICS_BASE_PATH}/startup/EpicsHostArch)
+ln -s ${EPICS_MODULES_PATH}/opcua/opcuaIocApp/libopcua.so ${EPICS_MODULES_PATH}/opcua/lib/${EPICS_HOST_ARCH}
+ln -s ${EPICS_MODULES_PATH}/opcua/opcuaIocApp/libopcua.so.0.9 ${EPICS_MODULES_PATH}/opcua/lib/${EPICS_HOST_ARCH}
```

Comment:
Do we need to keep both links? `libopcua.so.0.9` is the actual binary and `libopcua.so` is already a symbolic link. I don't know if we are ever going to need to support multiple versions to coexist.

Still, I think this can be simplified if we use [bash brace expansion](https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html):

```bash
ln -s ${EPICS_MODULES_PATH}/opcua/{opcuaIocApp/libopcua.so.0.9,lib/$EPICS_HOST_ARCH/libopcua.so}
```